app/routes/search_label.js

```diff
@@ -2,6 +2,14 @@ import AuthorizeRoute from './authorize';
 
 export default AuthorizeRoute.extend({
 
+  afterModel(model, transition){
+    var pkgs = model.get("packages");
+    if(pkgs && pkgs.length > 0 && (pkgs.get("firstObject.hasAllPackagesDesignated") || pkgs.get("firstObject.hasAllPackagesDispatched"))){
+      transition.abort();
+      this.get('messageBox').alert(this.get("i18n").t('designated_dispatched_error'));
```

Comment:
check this
https://jira.crossroads.org.hk/browse/GCW-1681?focusedCommentId=26664&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-26664